docs/3rd_party/pdf/snippets/pdfminer/extract_metadata.py

```diff
@@ -0,0 +1,32 @@
+from pdfminer.pdfdocument import PDFDocument
+from pdfminer.pdfparser import PDFParser
+
+
+def extract_pdf_metadata(pdf_file):
+    """
+    Extracts metadata from a PDF file using PDFMiner.six.
+
+    Args:
+    - pdf_file (str): Path to the PDF file.
+
+    Returns:
+    - metadata (dict): Dictionary containing PDF metadata.
+    """
```

Comment:
```suggestion
def extract_pdf_metadata(pdf_file: str) -> dict[str, str]:
    """
    Extracts metadata from a PDF file using `PDFMiner.six`.

    Args:
        pdf_file: Path to the PDF file.

    Returns:
        A dictionary containing PDF metadata.
    """
```

- The type annotation might be enough so no need to redundantly put types (I personally made a mistake in the browser example, will correct soon)
- The `-` seems extra according to the [style](https://google.github.io/styleguide/pyguide.html)

[Reference](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)